stable/insights-agent/values.yaml

```diff
@@ -396,9 +396,26 @@ rightsizer:
   # This image is for the controller, the agent only runs the Insights uploader.
   image:
     repository: quay.io/fairwinds/right-sizer
-    tag: 0.1.0
+    tag: 0.1.1
     pullPolicy: Always
+  imagePullSecrets:
   stateconfigmapname: insights-agent-right-sizer-controller-state
+  # rightsizer.resetOOMsWindow -- The amount of time after which, if no OOMs have been seen, a resource will be removed from this Insights report.
+  resetOOMsWindow: "24h"
+  # rightsizer.namespaces -- Only operate in these Kubernetes namespaces.
+  # By default, all namespaces are allowed.
+  # This applies both to alerting on OOM-kills, and updating memory limits (if enabled).
+  namespaces:
+  updateMemoryLimits:
+    # rightsizer.updateMemoryLimits.enabled -- Update memory limibs of resources whos containers are OOM-killed.
```

Comment:
we don't use helm-docs for this chart - you'll need to add these to the readme manually